packages/react-core/src/core/providers/thirdweb-wallet-provider.tsx

```diff
@@ -41,35 +41,47 @@ type WalletConnectParams<I extends WalletInstance> = Parameters<
 
 type ConnectionStatus = "unknown" | "connected" | "disconnected" | "connecting";
 
-type ConnectFnArgs<I extends WalletInstance> =
+type ConnectFnArgs<
+  I extends WalletInstance,
+  Config extends Record<string, any> | undefined = undefined,
```

Comment:
Why this type? `undefined = undefined` ?

Reply:
`= undefined` is the default

Reply:
`= undefined` is not part of type - it's the default. 
This is not required here - so removing it

Reply:
duh!! oof, my bad hehe